src/GovUk.Education.ExploreEducationStatistics.Content.ViewModels/DataGuidanceViewModel.cs

```diff
@@ -1,13 +1,15 @@
+using System.Diagnostics.CodeAnalysis;
 using GovUk.Education.ExploreEducationStatistics.Data.ViewModels;
 
 namespace GovUk.Education.ExploreEducationStatistics.Content.ViewModels;
 
 public record DataGuidanceViewModel : ReleaseSummaryViewModel
 {
-    public string DataGuidance { get;  }
+    public required string DataGuidance { get; init; }
 
-    public List<DataGuidanceDataSetViewModel> DataSets { get; }
+    public required List<DataGuidanceDataSetViewModel> DataSets { get; init; }
 
+    [SetsRequiredMembers]
```

Comment:
Might be good to communicate use of this new attribute to the rest of the team whenever there's a suitable time! 🙂

Reply:
Yep! Although one thing to note - it doesn't provide a compiler error if the required properties aren't filled out by the constructor. Rider will highlight this as a potential inspection warning, but I don't think the compiler will.

Something to be aware of when changing properties on anything that uses these types of constructors.

Reply:
Cool, just tried this out and I saw the warning in Rider you mentioned. I will bear this in mind, thanks.